pages/cpm.vue

```diff
@@ -69,14 +87,41 @@ export default {
       if (!L.Browser.ie && !L.Browser.opera && !L.Browser.edge) {
         layer.bringToFront()
       }
+
+      this.info.update(layer._leaflet_id)
+      console.log(layer._leaflet_id)
     },
     resetHighlight (e) {
       const layer = e.target
       layer.setStyle({ fillColor: '#ffffff', weight: 2, opacity: 1, color: 'white', dashArray: '3', fillOpacity: 0.7 })
+      this.info.update()
     },
     zoomToFeature (e) {
       this.map.fitBounds(e.target.getBounds())
+    },
+    infoDiv (map) {
+      this._div = L.DomUtil.create('div', 'info') // create a div with a class "info"
+      // this.update()
+      return this._div
+    },
+    updateInfo (props) {
+      this._div.innerHTML = '<h4>CPM model&data availability</h4>' + (this.groups[props] ? this.groups[props] : 'Hover over a region')
     }
   }
 }
 </script>
+
+<style>
+.info {
+    padding: 6px 8px;
+    font: 14px/16px Arial, Helvetica, sans-serif;
+    background: white;
+    background: rgba(255,255,255,0.8);
+    box-shadow: 0 0 15px rgba(0,0,0,0.2);
+    border-radius: 5px;
+}
+.info h4 {
+    margin: 0 0 5px;
+    color: #777;
+}
+</style>
```

Comment:
Can we use tailwind classes instead of pure css (eventually)? Or are these so specific that you cannot use tailwind here?

Reply:
I plan to take care of the styling in a separate PR. So just did a simple copy/paste from the example. I agree that we'd better eventually replace this with tailwind.